Source/WebCore/rendering/RenderBlock.cpp

```diff
@@ -1120,6 +1129,11 @@ void RenderBlock::paint(PaintInfo& paintInfo, const LayoutPoint& paintOffset)
     if (pushedClip)
         popContentsClip(paintInfo, phase, adjustedPaintOffset);
 
+    if (paintInfo.phase == PaintPhase::Foreground && !renderCaretInsideContentsClip()) {
```

Comment:
@smfr I want to render the caret outside of the RenderBlock / text field's clipping rectangle. Is there a better way to do this than ignoring the clipping rectangle? Should I paint the caret as part of a different paint phase?